io/eolearn/tests/test_processing_io.py

```diff
@@ -0,0 +1,105 @@
+""" Testing SentinelHubProcessingInput
+"""
+
+import unittest
+import datetime as dt
+from sentinelhub import CRS, BBox, DataSource
+
+from eolearn.io import SentinelHubProcessingInput
+from eolearn.core import FeatureType
+
+# import sys
+# import logging
+# logging.basicConfig(stream=sys.stdout)
+# logging.getLogger("eolearn.io.processing_api").setLevel(logging.DEBUG)
+# logging.getLogger("sentinelhub.sentinelhub_client").setLevel(logging.DEBUG)
+# logging.getLogger("sentinelhub.sentinelhub_rate_limit").setLevel(logging.DEBUG)
+
+class TestProcessingIO(unittest.TestCase):
+    """ Test cases for SentinelHubProcessingInput
+    """
+    size = (100, 100)
```

Comment:
minor comment that can be ignored

Sometimes I find it useful to test images with different height/width, to check that they are handled correctly throughout the code, so I would try (99, 101) to check they end up in correct dimensions

Reply:
Good one!

Reply:
Done.

Reply:
Did it.